src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -352,6 +352,21 @@ private static bool TryGetTextForKeyword(SyntaxToken token, ISyntaxFactsService
                 }
             }
 
+            if (token.IsKind(SyntaxKind.DefaultKeyword))
+            {
+                if (token.Parent is DefaultSwitchLabelSyntax)
+                {
+                    text = Keyword("defaultcase");
+                    return true;
+                }
+
+                if (token.Parent is DefaultExpressionSyntax or LiteralExpressionSyntax)
+                {
+                    text = Keyword("defaultvalue");
+                    return true;
+                }
```

Comment:
Is this check adding any value? Why not just let this fall through and use `default_CSharpKeyword` as it currently does? Seems to me like switch labels are the special case here.

Reply:
> Is this check adding any value? Why not just let this fall through and use `default_CSharpKeyword` as it currently does? Seems to me like switch labels are the special case here.
> 
> This is pending a decision on what happens in the docs PR of course.

Falling back to `default_CSharpKeyword` will lead to maintaining *less* f1_keywords in docs. So, +1 for that.

Reply:
I like the idea of falling back to the old `default_CSharpKeyword`, especially now that it no longer is used for the "choose which context" default page.

Reply:
Going this route would require moving the `default_CSharpKeyword` from the `/keywords/default.md` page to the `/operators/default.md` page.

Reply:
The main reason I originally left it in was in case any future usage of `default` appeared, as it would at least still route to the keyword page.  For now though, I'm opting to reuse the existing `default_CSharpKeyword` and move it to the appropriate page.  Does this present any backwards compatibility issues @Youssef1313?

Reply:
> The main reason I originally left it in was in case any future usage of `default` appeared, as it would at least still route to the keyword page. For now though, I'm opting to reuse the existing `default_CSharpKeyword` and move it to the appropriate page. Does this present any backwards compatibility issues @Youssef1313?

Changes in F1 area in Roslyn will *not* introduce any backcompat issues.
Changes in docs *may* introduce backcompat issues *if and only if* existing f1_keywords are completely deleted.

If a a future usage of `default` appear, that's not a problem, we can simply update the f1 service and introduce a new f1_keyword for it.